storage/plugins/archives/ArchivesPlugin.php

```
@@ -3983,6 +4035,858 @@ public function plannedHooks(): array
+        // OAI-PMH allows parameters via GET or POST (application/x-www-form-urlencoded).
+        $params = $request->getQueryParams();
+        if ($request->getMethod() === 'POST') {
+            $body = (array) ($request->getParsedBody() ?? []);
+            $params = array_merge($params, $body);
+        }
+
+        $verb   = (string) ($params['verb'] ?? '');
+        $now    = gmdate('Y-m-d\TH:i:s\Z');
+        $baseUrl = absoluteUrl('/archives/oai');
+
+        $xw = new \XMLWriter();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '4486', column '19'). (undefined)

(MissingImport)